src/utils/createFetchRecordFunctionFromUrl.ts

```diff
@@ -0,0 +1,9 @@
+export const createFetchRecordFunctionFromUrl = (url: string) => {
+  return (async ()=>{
+    const group = await fetch(url, {
```

Comment:
I think `group` name should be changed to sth like `results` and `parsedResults` or `data` below